### PR TITLE
Fixes #4713 Update docs about Python 3

### LINKED ISSUE #4713: Update docs about Python 3
The project is now working on Python3 and incompatible with Python2. Docs should be updated with this information

**Would you like to work on this?**

No

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #4713 Update docs about Python 3

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Updating docs about Python 3

#### Changes proposed in this pull request:

- Updating docs about Python 3
-
-
@iamareebjamal Please review thanks.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,7 @@ app.db
 .idea/
 .bundle/
 .vscode/
+venv/
```
Comment (iamareebjamal) [inline .gitignore]: If adding this, also add .venv

**Comment by iamareebjamal** [comment]
Python 3.6.4 but any version above Python 3.5 should work IMO

**Comment by mayank8318** [comment]
Anyone facing this error when running `pip3 install -r requirements.txt` in a conda env with python 3.6 ?
screenshot from 2018-05-13 15-11-08

**Comment by bhaveshAn** [comment]
@mayank8318 `librabbitmq` has been removed in the last commit.

**Pull request merged**